torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1226,6 +1228,60 @@ class IrParser {
       }
     }
 
+    {
+      auto ptr_op = getOperatorForLiteral(
+          "aten::autocast_to_fp16(Tensor(a) self) -> Tensor(a)");
+      registerParseRule(
+          ptr_op,
+          [](const Node* node,
+             std::unordered_map<size_t, CgValue>& value_map) -> void {
+            auto self = value_map[node->input()->unique()];
+            auto out = unaryOp(UnaryOpType::Set, self);
+            value_map.emplace(node->output()->unique(), out);
```

Comment:
Note to myself: I'll put a TODO here and open an issue to track the potential divergent aliasing behavior, comparing to execution in native PyTorch.

Reply:
tracked in #929